package.json

```diff
@@ -34,6 +34,7 @@
     "@angular/platform-browser": "2.3.0",
     "babel-cli": "^6.18.0",
     "babel-plugin-transform-react-jsx": "^6.8.0",
+    "babel-preset-es2015-rollup": "^3.0.0",
```

Comment:
Sounds good. It would be better for people who use Rollup and Babel if there wasn't a separate preset they had to worry about (es2015-rollup dates from a time before `modules: false`), so the more we can coalesce around a standard way of doing things that doesn't depend on specific forks of specific presets the better